src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -24,37 +40,22 @@ Flutter supports the following platforms:
 All channels include master, beta,
 and stable channels.
 
-## How we define a supported platform
-
-We define three tiers of support for the platforms on
-which Flutter runs:
-
-1. Supported Google-tested platforms
-  are automatically tested at every commit
-  by continuous integration testing.
-1. Best-effort platforms, supported through community
-   testing, are platforms we believe we support through
-   coding practices and ad-hoc testing,
-   but rely on the community for testing.
-1. Unsupported platforms, which are platforms that
-   might work, but that the development team
-   doesn't directly test or support.
-
-### Supported Google-tested platforms
+### Google-tested platforms
 
 |Platform|Version               |
 |--------|----------------------|
 |Android |Android SDK 19–30*    |
 |iOS     |14-15                 |
 |Linux   |Debian 10             |
-|macOS   |El Capitan & greater  |
+|Linux   |Ubuntu 18.04 LTS      |
+|macOS   |Monterey (12) & above |
 |Web     |Chrome 84             |
 |Web     |Firefox 72.0          |
 |Web     |Safari / Catalina     |
 |Web     |Edge 1.2.0            |
 |Windows |Windows 10            |
 
-* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
+\* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
```

Comment:
Is this change intentional?

Reply:
Yes I am escaping the * character referenced above